web/styles/subscriptions.css

```diff
@@ -863,6 +863,14 @@ h4.user_group_setting_subsection_title {
                 margin-right: 3px;
                 text-decoration: none;
             }
+
+            .deactivate {
+                height: 100%;
+
+                .icon-container {
+                    display: flex;
+                }
+            }
```

Comment:
These changes were made because `font-family: 'zulip-icons' !important;` caused the icon to be misaligned vertically. This update fixes the alignment issue. A better solution might avoid this kind of workaround, but it would require major changes to the `display` and other properties of the child elements in `stream_settings_header`.